server/api.go

```diff
@@ -534,3 +534,19 @@ func (p *Plugin) ServeHTTP(_ *plugin.Context, w http.ResponseWriter, r *http.Req
 
 	p.apiRouter.ServeHTTP(w, r)
 }
+
+func (p *Plugin) handleGetStats(w http.ResponseWriter) error {
+	stats, err := p.store.GetCallsStats()
+	if err != nil {
+		return fmt.Errorf("failed to get stats from store: %w", err)
+	}
+
+	// TODO: consider implementing some caching for heaviest queries
```

Comment:
Would you consider making it a ticket? (or did you mean to clean this up before submitting?)

Reply:
Yes, I can make a ticket. I just don't know yet if this is really needed. I almost feel it may be best to go with https://github.com/mattermost/mattermost-plugin-calls/pull/762#discussion_r1626483383.